modular_skyrat/modules/modular_weapons/code/automatic.dm

```diff
@@ -51,7 +51,9 @@
 	fire_sound = 'sound/weapons/gun/smg/shot_alt.ogg'
 	weapon_weight = WEAPON_MEDIUM
```

Comment:
this is supposed to be full auto, the burst fire was a secondary option (as burst is much-much stronger)